src/main/java/ratpack/zipkin/ServerRequest.java

```diff
@@ -0,0 +1,36 @@
+package ratpack.zipkin;
+
+import ratpack.http.Headers;
+import ratpack.http.HttpMethod;
+
+/**
+ * Read-only API for wrappers around Ratpack's {@link ratpack.http.Request}.
+ */
+public interface ServerRequest {
```

Comment:
Strictly speaking, this isn't necessary because Ratpack's `Request` gives us access to everything we would need. But I've kept this around because it felt like a read-only API in the lambda sort of communicates better what you can/cannot do (i.e don't try to mutate things in the lambda). Also for some reason it felt weird to have the lambda take a Ratpack `Request` for one param and a `ServerResponse` for the other.